src/main/scala/com/raphtory/core/components/partition/Reader.scala

```diff
@@ -62,6 +63,7 @@ class Reader(
         val jobID         = req.jobID
         val queryExecutor = new QueryExecutor(partitionID, storage, jobID, conf, pulsarController)
         scheduler.execute(queryExecutor)
+        PartitionTelemetry.queryExecutorMapCounter.inc()
```

Comment:
Is this decremented when a job finishes?

Reply:
Yes, this is being decremented when the job finishes